api/v1/controllers/samples.js

```diff
@@ -62,7 +150,35 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   getSample(req, res, next) {
-    doGet(req, res, next, helper);
+    if (featureToggles.isFeatureEnabled(constants.featureName)) {
+      const sampleName = req.swagger.params.key.value.toLowerCase();
+      const aspectName = sampleName.split('|')[ONE];
+      const commands = [];
+
+      // get sample
+      commands.push([
+        constants.commands.hgetall,
+        sampleStore.toKey(constants.objectType.sample, sampleName),
+      ]);
+
+      // get aspect
+      commands.push([
+        constants.commands.hgetall,
+        sampleStore.toKey(constants.objectType.aspect, aspectName),
+      ]);
+
+      redisClient.batch(commands).execAsync()
+      .then((responses) => {
+        // clean and attach aspect to sample
+        const sampleRes = cleanAddAspectToSample(
+          responses[ZERO], responses[ONE], res.method
+        );
+
+        res.status(httpStatus.OK).json(sampleRes);
+      });
+    } else {
+      doGet(req, res, next, helper);
+    }
   },
 
```

Comment:
I am using nouns for now, for the lack of a better word to use. Maybe verb" is better but the more changes I am doing now I am leaning towards "models". 
@iamigo Any suggestions?